torch/_inductor/ir.py

```diff
@@ -2789,11 +2789,11 @@ def __str__(self):
 
     __repr__ = __str__
 
-    def is_contiguous(self):
+    def is_contiguous(self) -> bool:
         return is_contiguous_strides_for_shape(self.stride, self.size)
 
     @staticmethod
-    def is_channels_last_contiguous(shape, strides):
+    def is_channels_last_contiguous(shape: Any, strides: Any) -> bool:
```

Comment:
It's hard to get perfect coverage but ideally a lot more the functions would be typed without `Any`.